internal/resources/cluster/resource_cluster_test.go

```diff
@@ -53,36 +53,79 @@ func TestAcceptanceForAttachClusterResource(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					checkResourceAttributes(provider, clusterConfig["attachWithKubeConfig"]...),
 				),
+				SkipFunc: func() (bool, error) {
```

Comment:
Can you add a validation in the test and fail the test if required env variables are not set clearly indicating which env variable they are missing like it is done in EKS cluster or git repository resources?

Reply:
Failing the test, will exit completely. 
Instead added a detailed log message and skipping it, so that next cluster variance test can run